docs/en/integrations/data-visualization/looker-studio-and-clickhouse.md

```diff
@@ -121,3 +121,13 @@ Finally, you should be able to see the introspected table structure and adjust t
 <br/>
 
 Now you can proceed with exploring your data or creating a new report! 
+
+## ClickcHouse Cloud Connection
+
+When using ClickHouse Cloud, you must first file a support case to enable the MySQL feature. Our support team will enable this feature, which then creates a dedicated user for the MySQL protocol.
+
+![Looker Studio Cloud Connection](./images/looker-studio-mysql-cloud.png)
+
+In the Looker Studio UI, choose the "Enable SSL" option. Upload the [LetsEncryptRoot Cert](https://letsencrypt.org/certificates/) as the server cert. ClickHouse Cloud's SSL certificate is signed by LetsEncrypt. You can download this root cert [here](https://letsencrypt.org/certs/isrgrootx1.pem).
```

Comment:
Do we need the first LetsEncryptRoot Cert link? We could just provide a link to download the pem.

Reply:
Let me reword. We can link the first link for the signed by LetsEncrypt. The first link is to comfort the users who are very sensitive about security. To give context why this cert, etc.

Reply:
updated.